activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/QueryProcessInstancesEntityIT.java

```diff
@@ -421,6 +425,65 @@ public void shouldGetProcessInstancesAsAdmin() {
                                          PAGED_PROCESS_INSTANCE_RESPONSE_TYPE);
     }
 
+    @Test
+    public void shouldGetProcessInstancesFilteredByStartDate() {
+        //given
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
+        sdf.setTimeZone(TimeZone.getTimeZone("GMT"));
+
+        Date startDate1 = new Date();
+        Date startDate2 = new Date();
+        Date startDate3 = new Date();
+        Calendar cal = Calendar.getInstance();
+        cal.set(Calendar.SECOND, 0);
+        cal.set(Calendar.MILLISECOND, 0);
+        Date now = cal.getTime();
+
+        //set start date as current date + 1
+        startDate1.setTime(now.getTime() + Duration.ofDays(1).toMillis());
+
+        ProcessInstance runningProcess1 = processInstanceBuilder
+            .aRunningProcessInstanceWithStartDate("first", startDate1);
+
+        startDate2.setTime(now.getTime() + Duration.ofDays(3).toMillis());
+        ProcessInstance runningProcess2 = processInstanceBuilder
+            .aRunningProcessInstanceWithStartDate("second", startDate2);
+
+        startDate3.setTime(now.getTime() - Duration.ofDays(3).toMillis());
+        ProcessInstance runningProcess3 = processInstanceBuilder
+            .aRunningProcessInstanceWithStartDate("third", startDate3);
```

Comment:
Suggestion: instead of using generic variable names such as `runningProcess1`, `runningProcess2`, `runningProcess3` you could names that help to understand what is their purpose such as `processStartedBeforePeriod`, `processStartedDuringPeriod`,`processStardedAfterPeriod` (feel free to use other names). Similarly, date varaibles could be `now`, `inThreeDays` and `threeDaysAgo`. What do you think?

Reply:
Makes sense.